src/scheduler/fifo.c

```diff
@@ -2355,6 +2351,46 @@ next_job(status *policy, server_info *sinfo, int flag)
 	return rjob;
 }
 
+/**
+ * @brief
+ *	Helper function used to copy the given source string to the destination string. It also
+ *	frees the destination and then allocates required memory before copying.
+ *
+ * @param[in] dest - Address of pointer to destination string
+ * @param[in] dest - pointer to source string
+ *
+ * @retval
+ * @return 0 - Failure
+ * @return  1 - Success
+ *
+ * @par Side Effects:
+ *	None
+ *
+ *
+ */
+static int
+copy_attr_value(char **dest, char *src)
```

Comment:
This function should be in misc.c, not fifo.c

Reply:
Moved this function to misc.c